cpp/include/raft/sparse/neighbors/detail/connect_components.cuh

```diff
@@ -287,6 +328,7 @@ template <typename value_idx, typename value_t>
 void min_components_by_color(raft::sparse::COO<value_t, value_idx>& coo,
                              const value_idx* out_index,
                              const value_idx* indices,
+                             const value_idx* sort_plan,
```

Comment:
Could you update the docstring of this function? It looks out of date (not just missing sort_plan)